homeassistant/components/romy/config_flow.py

```diff
@@ -0,0 +1,151 @@
+"""Config flow for ROMY integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import romy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import zeroconf
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD, CONF_UUID
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, LOGGER
+
+
+def _schema_with_host() -> vol.Schema:
+    return vol.Schema(
+        {
+            vol.Required(CONF_HOST): cv.string,
+        },
+    )
+
+
+def _schema_with_password() -> vol.Schema:
+    return vol.Schema(
+        {vol.Required(CONF_PASSWORD): vol.All(cv.string, vol.Length(8))},
+    )
+
+
+class RomyConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle config flow for ROMY."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self) -> None:
+        """Handle a config flow for ROMY."""
+        self.romy_info: dict[str, Any] = {}
+
+    async def async_step_user(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Handle the user step."""
+        errors: dict[str, str] = {}
+        data = _schema_with_host()
+
+        if user_input:
+            self.romy_info[CONF_HOST] = user_input[CONF_HOST]
+
+            new_romy = await romy.create_romy(self.romy_info[CONF_HOST], "")
+
+            if not new_romy.is_initialized:
+                errors[CONF_HOST] = "cannot_connect"
+            else:
+                # check if already setuped
```

Comment:
There is a typo/grammar issue here here, but i think the comment can just be removed
```suggestion
```

Reply:
removed